3.2/Dockerfile

```diff
@@ -33,26 +30,25 @@ RUN yum install -y centos-release-scl && \
     chmod g+w -R /var/opt/rh/rh-mongodb32/lib/mongodb && \
     chmod -R g+rwx /var/lib/mongodb
 
-# Get prefix path and path to scripts rather than hard-code them in scripts
-ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
+ENV MONGODB_VERSION=3.2 \
+    HOME=/var/lib/mongodb \
+    # Set paths to avoid hard-coding them in scripts.
+    CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
```

Comment:
why not put all the ENV definitions first, instead of after the RUN, if you want to optimize cache reuse?  it also puts the "MONGODB_VERSION" variable front and center so it doesn't get overlooked during updates.